lua/cinema/theater/services/sh_alexjonesnetwork.lua

```diff
@@ -1,18 +1,19 @@
 ﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
-local SERVICE = {}
-SERVICE.Name = "AlexJonesNetwork"
-SERVICE.NeedsCodecs = true
-SERVICE.NeedsChromium = true
-local rumbleService = theater.GetServiceByClass("rumble")
+local AJNSERVICE = {}
```

Comment:
`SERVICE`->`AJNSERVICE` because I think it was conflicting with `SERVICE` in `sh_rumble.lua`